test/exchange/ExchangeTests.js

```diff
@@ -60,17 +60,17 @@ describe('Exchange Contract', () => {
     // Asset 1 has 200 basis points towards creator 1
     // Asset 2 has 200 basis points towards creator 1, 100 basis points towards creator 2
     var asset = [
-      [1, "arweave.net/tx/public-uri-1", "arweave.net/tx/private-uri-1", ethers.constants.MaxUint256, creator1Address.address, 20000],
-      [2, "arweave.net/tx/public-uri-2", "arweave.net/tx/private-uri-2", 100, ethers.constants.AddressZero, 0],
+      ["arweave.net/tx/public-uri-1", "arweave.net/tx/private-uri-1", ethers.constants.MaxUint256, creator1Address.address, 20000],
```

Comment:
It doesn't really affect anything, but I guess to be more consistent with the other contracts the uri numbers should be decremented by one.